cours.json

```diff
@@ -95,6 +95,22 @@
       "containers/conclusion"
     ]
   },
+  "kubernetes-user": {
+    "course_name": "kubernetes-user",
+    "course_description": "Formation Kubernetes Utilisateur",
+    "modules": [
+      "intro-osones",
+      "containers/kubernetes/intro-osones",
+      "containers/kubernetes/k8s-governance",
+      "containers/kubernetes/k8s-architecture",
```

Comment:
is that really for a *user* training?

Reply:
We'll all discuss about the list of modules inside the kubernetes user/ops training